src/main/java/seedu/address/model/person/Person.java

```diff
@@ -110,7 +116,8 @@ public String toString() {
                 .append("; Lesson Plan: ")
                 .append(getLessonPlan())
                 .append("; Homework: ")
-                .append(getHomeworkList());
+                .append(getHomeworkList())
+                .append(getAttendanceList());
```

Comment:
Person::equals should equate the AttendanceLists also

Reply:
noted changed with thanks